client/state/reader/streams/reducer.js

```diff
@@ -46,7 +45,7 @@ export const items = ( state = [], action ) => {
 				const afterGap = takeRightWhile( state, ( postKey ) => ! keysAreEqual( postKey, gap ) );
 
 				// after query param is inclusive, so we need to drop duplicate post
-				if ( keysAreEqual( last( streamItems ), afterGap[ 0 ] ) ) {
+				if ( keysAreEqual( streamItems?.at( -1 ), afterGap[ 0 ] ) ) {
```

Comment:
`.at()` is relatively new and the browser support isn't particularly clear for some browsers:

https://caniuse.com/mdn-javascript_builtins_array_at

At the same time, I've seen some recent Sentry errors that contain "`.at()` is not a function" which might be a sign that there are some of those browser. Looking at Sentry, it seems that most of them are older Safari versions that don't support it yet (< 15.3). 

While we officially support only the latest 2 browser versions, it's easy enough but safer to do `[ ...streamItems ].pop()` or `streamItems?.[ streamItems.length - 1 ]` (the latter seems better to preserve the optional chaining)

While I'm fine with `Array.prototype.at()` in general, it might make sense to go with the safer version.

Reply:
That's a fair point 👍🏻 I think `.at` is most descriptive but I don't see a reason not to use one of the alternatives